site/src/pages/release-notes/1.27.0.mdx

```diff
@@ -0,0 +1,228 @@
+---
+date: 2024-01-28
+---
+
+## 🌟 New features
+
+- **xDS-based EndpointGroup**: You can utilize xDS protocols to create <type://XdsEndpointGroup> for service
```

Comment:
How about `xDS support`?
How about also adding a link to it? https://www.envoyproxy.io/docs/envoy/latest/api-docs/xds_protocol

Reply:
> How about xDS support?

- We don't fully support xDS protocol so the scope is limited to `EndpointGroup`.

> How about also adding a link to it? 

I'm late. I was about to add it.